src/main.js

```diff
@@ -2,9 +2,11 @@ import { createApp } from "vue";
 import App from "./App.vue";
 import router from "./router";
 import MainButton from "./components/MainButton/MainButton.vue";
+import UserLogin from "./components/UserLogin/UserLogin.vue";
 
-const app = createApp(App).use(router);
 
+const app = createApp(App).use(router);
 app.component("main-button", MainButton);
+app.component("user-login", UserLogin);
```

Comment:
Why it has to be a global component? Please refactor it to explicit imports where it's used

Reply:
fixed